eth/pubsub.go

```diff
@@ -165,6 +168,65 @@ func (p *PubSub) handleBeaconBlock(ctx context.Context, msg *pubsub.Message) err
 	return nil
 }
 
+func (p *PubSub) handleBlobSidecar(ctx context.Context, msg *pubsub.Message) error {
+	switch p.cfg.ForkVersion {
+	case DenebForkVersion:
+		var blob ethtypes.BlobSidecar
+		err := p.cfg.Encoder.DecodeGossip(msg.Data, &blob)
+		if err != nil {
+			slog.Error("decode blob sidecar gossip message", tele.LogAttrError(err))
+			return err
+		}
+
+		slot := blob.GetSignedBlockHeader().GetHeader().GetSlot()
+		slotStart := p.cfg.GenesisTime.Add(time.Duration(slot) * p.cfg.SecondsPerSlot)
+		proposerIndex := blob.GetSignedBlockHeader().GetHeader().GetProposerIndex()
+
+		now := time.Now()
+		evt := &host.TraceEvent{
+			Type:      "HANDLE_MESSAGE",
+			PeerID:    p.host.ID(),
+			Timestamp: now,
+			Payload: map[string]any{
+				"PeerID":     msg.ReceivedFrom.String(),
+				"MsgID":      hex.EncodeToString([]byte(msg.ID)),
+				"MsgSize":    len(msg.Data),
+				"Topic":      msg.GetTopic(),
+				"Seq":        msg.GetSeqno(),
+				"Slot":       slot,
+				"ValIdx":     proposerIndex,
+				"TimeInSlot": now.Sub(slotStart).Seconds(),
+				"StateRoot":  hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetStateRoot()),
+				"BodyRoot":   hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetBodyRoot()),
+				"ParentRoot": hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetParentRoot()),
+			},
+		}
+		slog.Info(
+			"Handling blob gossip message",
+			"PeerID", p.host.ID(),
+			"RemotePeerID", msg.ReceivedFrom.String(),
+			"MsgID", hex.EncodeToString([]byte(msg.ID)),
+			"MsgSize", len(msg.Data),
+			"Topic", msg.GetTopic(),
+			"Seq", msg.GetSeqno(),
+			"Slot", slot,
+			"ValIdx", proposerIndex,
+			"TimeInSlot", now.Sub(slotStart).Seconds(),
+			"StateRoot", hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetStateRoot()),
+			"BodyRoot", hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetBodyRoot()),
+			"ParentRoot", hexutil.Encode(blob.GetSignedBlockHeader().GetHeader().GetParentRoot()),
+		)
```

Comment:
I don't understand why the `slog` logging is required, this information is duplicate with the `evt`.

Reply:
It's not necessarily "required", but helps a little in terms of debugging and showing what's coming from those topics.

Removed it for now, and if we need them in the future, we can add back with Debug or Trace level logging.